CHANGELOG.md

```diff
@@ -94,6 +94,7 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 * (simulation) [#18196](https://github.com/cosmos/cosmos-sdk/pull/18196) Fix the problem of `validator set is empty after InitGenesis` in simulation test.
 * (baseapp) [#18551](https://github.com/cosmos/cosmos-sdk/pull/18551) Fix SelectTxForProposal the calculation method of tx bytes size is inconsistent with CometBFT
 * (server) [#18994](https://github.com/cosmos/cosmos-sdk/pull/18994) Update server context directly rather than a reference to a sub-object
+* (upgrade) [#19706](https://github.com/cosmos/cosmos-sdk/pull/19706) Stop treating inline JSON as a URL.
```

Comment:
Can you move this changelog to x/upgrade?

Reply:
Done.

Reply:
I don't see it 😅

Reply:
Oh, I see what you understood.
I actually meant to move this changelog entry to x/upgrade/changelog.md, as modules now have their own changelog.

Reply:
Ah, I see. Done.